Detectors/FIT/FT0/base/include/FT0Base/Geometry.h

```diff
@@ -65,9 +65,15 @@ class Geometry
     return o2::base::GeometryManager::getPNEntry(getDetID(), index);
   }
 
+  // Calculate the coordinates of all the channels.
+  void calculateChannelCenter();
+  // Get the coordinates of the center of the channel channelId.
+  TVector3 getChannelCenter(UInt_t channelId) { return mChannelCenter[channelId]; }
```

Comment:
please put `const`

Reply:
since this returns a copy in any case I don't think `const` makes a difference, but up to you of course the way you want to use this function

Reply:
I have added const for integrity of the system.